nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -152,7 +152,8 @@ language governing permissions and limitations under the License. -->
         </dependency>
         <dependency>
             <groupId>org.apache.activemq</groupId>
-            <artifactId>activemq-client</artifactId>
+            <artifactId>activemq-all</artifactId>
+            <scope>test</scope>
```

Comment:
@olegz - this is removing the activemq-client from the runtime classpath. Was that the intent? Is it already brought in elsewhere, or am I missing something?